src/test/java/ibm/jceplus/junit/base/BaseTestAESCopySafe.java

```diff
@@ -32,27 +33,17 @@ public class BaseTestAESCopySafe extends BaseTest {
 
     enum MODE { CBC, GCM };
 
-    protected int specifiedKeySize = 0;
-
-    public BaseTestAESCopySafe(String providerName) {
-        super(providerName);
-        this.specifiedKeySize = 128;
-    }
-
-    public BaseTestAESCopySafe(String providerName, int keySize) throws Exception {
-        super(providerName);
-        this.specifiedKeySize = keySize;
-        Assume.assumeTrue(javax.crypto.Cipher.getMaxAllowedKeyLength("AES") >= keySize);
```

Comment:
Should we keep the keysize check?

Reply:
I added this assumption back in. In general this test is not configuring different sized keys in the tests but instead always makes use of a key size of 128. At the time i removed this assumption check since it seemed redundant and stuck out from our other pattern of code here but i added it back in again as it shouldn't hurt anything.